charm-slurmctld/config.yaml

```diff
@@ -8,9 +8,9 @@ options:
       This can be set to the Organization's locall mirror/cache of packages and
       superseeds the Omnivector repositories. Alternatively, it can be used to
       track a `testing` Slurm version, e.g. by setting to
-      `ppa:omnivector/osd-testing`.
-
-      Note: currently supported for Ubuntu only.
+      `ppa:omnivector/osd-testing` (on Ubuntu), or
```

Comment:
We should make a note that this configuration is only applicable post deploy. If the user changes this value after initially deploying the charms then they will have inconsistent versions.

Reply:
The advantage of having it in each charm is that it allows the administrator to update/change different daemons. For example: if one wants to update only `slurmdbd`, they can have a specific URL in the `slurmdbd` charm and that would not impact any other charm.